internal/pkg/aws/ecs/ecs.go

```diff
@@ -168,6 +181,60 @@ func (e *ECS) ServiceTasks(clusterName, serviceName string) ([]*Task, error) {
 	return tasks, nil
 }
 
+// DefaultClusters returns the default clusters in the account and region
+func (e *ECS) DefaultClusters() ([]string, error) {
+	resp, err := e.client.DescribeClusters(&ecs.DescribeClustersInput{})
+	if err != nil {
+		return nil, fmt.Errorf("get default clusters: %w", err)
+	}
+
+	if len(resp.Clusters) == 0 {
+		return nil, errors.New("no default cluster is found")
+	}
+
+	clusters := make([]string, len(resp.Clusters))
+	for idx, cluster := range resp.Clusters {
+		clusters[idx] = aws.StringValue(cluster.ClusterArn)
+	}
+
+	return clusters, nil
```

Comment:
From what I've known, it is not possible. Right now the default cluster is simply the cluster that's named "default". Since clusters can't have the same name, the default cluster must be unique. 

My concern was that perhaps in the future they will identify the default cluster by tags or attributes, and as a result multiple default clusters would be possible.

On a second thought, I agree that it makes more sense to return only one cluster from `DefaultCluster`. In the (unlikely) event that multiple default clusters are possible in the future, it is easier for us to locate where we need to make the changes. I will update the code accordingly!

Reply:
It's more like a question rather than a strong inclination for code change. Thanks for the deep investigation! I'm ok for both.